### ISSUE TITLE
Can't use together with expressions plugin

### ISSUE THREAD

**Issue by cossssmin** [posthtml/posthtml-fetch#2]
Found a weird issue: if you use `posthtml-expressions` together with this plugin, it will fail with either a `TypeError` or a `ReferenceError` (depending on which one you add to the plugins array first).

So this setup:

```js
const html = `<fetch url="https://jsonplaceholder.typicode.com/users/1">
  {{ response.name }}
</fetch>`

posthtml([
    require('posthtml-expressions')({ locals: { var: 'test' } }),
    require('posthtml-fetch')()
  ])
    .process(html)
    .then(result => result.html)
    .catch(error => { throw error })
```

Fails with `ReferenceError: response is not defined`

If you switch plugin order, you get `TypeError: nodes.reduce is not a function`.

_However_, if you skip expression parsing (with either `@{{ }}` or a `<raw>` tag), it works 😵

```html
<fetch url="https://jsonplaceholder.typicode.com/users/1">
  @{{ response.name }}
</fetch>
```

... that correctly outputs 'Leanne Graham' and the build doesn't break.

@Scrum any ideas? Is my setup wrong or is there a bug in the plugin?

**Comment by Scrum** [comment]
@cossssmin Hi, I understand it like this:
1. If used `@{{ response.name }}` then the value is taken from the local scoup
2. If used `{{ response.name }}` then the value is taken from the global scoup
and trying to find a variable in the global scope `({ locals: { var: 'test' } })` it fails

**Comment by cossssmin** [comment]
Yes, that's what I also understand is happening... But it shouldn't, and in fact you should be able to use it _before_ `posthtml-expressions` like this:

```js
posthtml([
    require('posthtml-fetch')()
    require('posthtml-expressions')({ locals: { var: 'test' } }),
  ])
    .process(html)
    .then(result => result.html)
```

But if you do that, you get an error in `posthtml-expressions` about `nodes.reduce`

**Comment by cossssmin** [comment]
Added a failing test in the [expressions](https://github.com/posthtml/posthtml-fetch/tree/expressions) branch. As you'll see, it currently fails with:

```sh
TypeError {
  message: 'nodes.reduce is not a function',
}
```
